### ISSUE TITLE
Pagination link does not meet accessibility compliant to screen reader users

### ISSUE THREAD

**Issue by ivanteoh** [eea/eea.facetednavigation#93]
Recently one of our 4.x plone sites was audited for WCAG 2.0 AA. The report came back and mentioned that pagination links in faceted navigation need to have hidden structural labels, 'Go to page:'. Below are explanation on 'hidden structural labels' from the agent.

Hidden structural labels are specific text for screen reader users only, which describe the content. Nested lists are hierarchical, but their hierarchy is not clear to screen reader users. Therefore you need to use a hidden structural label to indicate the beginning and end of sub-navigation. For code examples see: http://maxdesign.com.au/articles/about-structural-labels/

For examples where hidden structural labels have been employed in navigation see:
http://www.adcq.qld.gov.au/about-us/the-commissioner (left-hand navigation)
http://www.melbourne.vic.gov.au/business/doing-business/Pages/doing-business-in-melbourne.aspx

**Comment by avoinea** [comment]
Pagination comes from Plone core via:
```
<div metal:use-macro="context/batch_macros/macros/navigation" />
```
Thus, you should fill a ticket to https://github.com/plone/plone.batching if this is still an issue.
